### ISSUE TITLE
Error pac.NetDeserializeTable attempt to call nil value

### ISSUE THREAD

**Issue by thegrb93** [Chessnut/NutScript#830]
https://github.com/CapsAdmin/pac3/issues/820

Pac plugin out of date.

**Comment by Hermit17** [comment]
Yeah I reinstalled Nutscript 1.1 and PAC3 (even tried the "PAC3 Github version" addon). Still getting the same error, going to see if I can grab the PAC plugin from a friend who's been using it for a server he was running.

**Comment by Hermit17** [comment]
Alright his works fine, I'll place the line of code that I believe has been changed as I can't actually give you the lua file. However whenever I clear PAC it reverts me back to my context menu playermodel instead of my set playermodel.

On line 59 under "-- Reject unauthorized PAC3 submits" and above "-- Get Player's PAC3 Parts."

net.Receive("pac_submit", function(_, ply)
		if (!ply) then return end -- ???
		if (nut.config.get("pacAdminOnly") and !ply:IsSuperAdmin()) then
			ply:notifyLocalized("illegalAccess")
			return 
		end

		local data = pace.net.DeserializeTable()
		pace.HandleReceivedData(ply, data)
	end)
end

**Comment by Hermit17** [comment]
Well my friend just told me to delete the entire plugin file and use the workshop addon. That also fixed almost everything, Context menu playermodel is still showing up when I clear my PAC but other than that it fixed it.
